packages/release/activate-preconfigured.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Isolates preconfigured.target
+DefaultDependencies=no
+After=fipscheck.target
+Requires=fipscheck.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/systemctl set-default preconfigured
+ExecStart=/usr/bin/systemctl isolate default --no-block
+RemainAfterExit=true
+StandardOutput=tty
+StandardError=inherit
```

Comment:
This is a common feature of the new units since `journald` isn't running yet when they run.